docker-app/qfieldcloud/core/serializers.py

```diff
@@ -180,6 +184,13 @@ def get_members(self, obj):
             ).values("member__username")
         ]
 
+    def get_teams(self, obj: Organization) -> list[str]:
+        """Implementation of `SerializerMethodField` for `teams`. Returns list of team names."""
+        return [
+            t.team_name
```

Comment:
```suggestion
            t.teamname
```

Note that the suggestions I make are not always typo-free/compilable. The property is actually `teamname` instead of `team_name`.